tasties_app/tests/conftest.py

```diff
@@ -183,3 +183,23 @@ def recipe():
 def signed_up_credentials():
     user = User.objects.create_user(username=VALID_USER, email=VALID_EMAIL, password=VALID_PASSWORD)
     user.save()
+
+
+@pytest.fixture
+def salad_recipes_with_rating():
```

Comment:
Please change the name of the fixture. The recipes no longer have ratings.

Reply:
The name of the fixture should describe what it does. So I would go for something like 'recipes_filtered_by_salad`.